test/bugs/ui-4071.js

```diff
@@ -69,7 +70,7 @@ test(
       headers: {
         'Content-Type': 'application/x-www-form-urlencoded',
       },
-      body: 'industries=1%2C16'
+      body: 'industries=1,16'
```

Comment:
Should `,` obey `allowReserved`?

Currently it follows the example table, regardless of `allowReserved`.
https://github.com/OAI/OpenAPI-Specification/blob/v3.1.0-dev/versions/3.1.0.md#encodingObject